pyproject.toml

```diff
@@ -99,5 +99,5 @@ target-version = "py311"
 
 [tool.ruff.lint.flake8-quotes]
 docstring-quotes = "double"  # https://docs.astral.sh/ruff/settings/#lint_flake8-quotes_docstring-quotes
-inline-quotes = "single"  # https://docs.astral.sh/ruff/settings/#lint_flake8-quotes_inline-quotes
+inline-quotes = "double"  # https://docs.astral.sh/ruff/settings/#lint_flake8-quotes_inline-quotes
```

Comment:
BTW, I don't want to make other people use double quotes if they're not used to or don't want to - so I was thinking more along the lines of turning this off.

Reply:
sure, we can remove the check altogether if ppl don't want to converge. But I do feel like specifically these format-only kind of rules, like whitespace, naming conventions, quotes, line-spilling style, etc. we could agree to be group-consistent (in our own stuff).